sprint_1/src/laura/uebungsprojekt/aufgaben/UPLK83_Pasch.java

```diff
@@ -0,0 +1,20 @@
+package laura.uebungsprojekt.aufgaben;
+
+import java.util.Random;
+
+public class UPLK83_Pasch {
+    public static boolean diceRoll(int dice_one, int dice_two, int dice_three) {
+        System.out.println("dice 1: " + dice_one + " dice 2: " + dice_two + " dice 3: " + dice_three);
+        if (dice_one == dice_two && dice_two == dice_three) {
+            return true;
+        } else return false;
```

Comment:
Das else brauchst du nicht und würde man glaube ich in der Regel auch weglassen. Können wir gerne später nochmal drüber sprechen.